### PR TITLE
Slovenia: admin updates and new features

### PR DESCRIPTION
**Pull request by stepps00**
Fixes https://github.com/whosonfirst-data/whosonfirst-data/issues/755.

This PR includes the following:

- **12** new region records. These records represent "statistical regions," so they were given a `mz:hierarchy_label` property of `0` and `wof:statistical_gore` property of `1`. These records were also given a `mz:note` of `"statistical region, no administrative function"`.
- **212** new `localadmin` records. These records supersede the existing 192 `county` records in this country.
- **192** existing `records` records were given a `edtf:cessation` property, an `mz:is_current` property of `0`, and a superseded link to the relevant new `localadmin` record that supersedes it.

### CODE REVIEW

File: data/110/895/959/5/1108959595.geojson [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    "geom:longitude":16.271221,
+    "iso:country":"SI",
+    "lbl:latitude":46.58822,
+    "lbl:longitude":16.269223,
+    "src:geom":"svn-sma",
+    "src:lbl:centroid":"mapshaper",
+    "svn-sma:svn-sma:d_od":"1995-01-01",
+    "svn-sma:svn-sma:dv_od":"1995-01-01",
+    "svn-sma:svn-sma:enota":"OB",
+    "svn-sma:svn-sma:ob_id":86,
```
Comment (nvkelso) [inline data/110/895/959/5/1108959595.geojson]: Fix duplicate property keys:

`"svn-sma:svn-sma:ob_id":86,`
to
"svn-sma:ob_id":86,

**Comment by nvkelso** [comment]
I looks like the new features don't always carry forward all the name and other properties from the old feature, when they should.

- old: [85677875](https://whosonfirst.mapzen.com/spelunker/id/85677875/#11/46.5286/16.1655) with lots of name variants.
- new: `1108959853` with no name variants

Suggest writing a general toolbox script to carry properties forward when the features are otherwise the "same".

- `name` prefixed properties
- `wof` prefixed properties
- concordances?
- anything else?

Seems easy when if there isn't already a value on the new feature from the old feature just copy it. When there are two disputes then skip for v1? Might need special treatment for `wof:population`?

**Comment by nvkelso** [comment]
@stepps00 Please also document `wof:statistical_gore` in https://github.com/whosonfirst/whosonfirst-properties/blob/master/properties/wof.md.

File: data/110/895/950/5/1108959505.geojson [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    "lbl:longitude":14.726389,
+    "src:geom":"svn-sma",
+    "src:lbl:centroid":"mapshaper",
+    "svn-sma:svn-sma:d_od":"1995-01-01",
+    "svn-sma:svn-sma:dv_od":"1995-01-01",
+    "svn-sma:svn-sma:enota":"OB",
+    "svn-sma:svn-sma:ob_id":67,
+    "svn-sma:svn-sma:ob_mid":11027075,
+    "svn-sma:svn-sma:ob_tip":"N",
+    "svn-sma:svn-sma:ob_uime":"Lue",
```
Comment (stefanb) [inline data/110/895/950/5/1108959505.geojson]: This should be "Luče", but "č" was lost somewhere.

**Comment by nvkelso** [comment]
Thanks for the tip!

**Comment by stepps00** [comment]
@stefanb - thank you! I will investigate and push changes.

**Comment by stepps00** [comment]
Most recent commits should address all change requests in this PR.

The problem was due to mapping that names straight to utf-8, when the following encoding/decoding should have taken place on all name fields:

Example:
```
new_name = row['_SR_UIME,C,50'].decode("Windows-1250").encode("utf8")
```

I will re-review before merge, but thank you to @stefanb for catching this before we added these records to Who's On First!

**Comment by stefanb** [comment]
Text encoding seems to be ok now - at a glance I didn't notice anything wrong anymore.
👍

**Comment by stepps00** [comment]
There are no longer merge conflicts in this PR, but there is additional work to be done to move properties (`ne:*` and `name:*`) from superseded records to the records that supersedes them.. holding off on merging until a new commit is made to address this issue.

**Comment by stepps00** [comment]
The most recent commit adds the `statoids:*` admin1 data to the twelve statistical regions in Slovenia.

**Comment by nvkelso** [comment]
Oops, @stepps00 for merge conflicts, then over to @thisisaaronland.

**Comment by stepps00** [comment]
Noting that duplicate hierarchies are occurring when PIP work is done on descendants because (some of?) the records with cessation dates are not actually tagged with `"mz:is_current":0`...

**Comment by stepps00** [comment]
Tagging @nvkelso for review before merge.

Most recent commit adds PIP work to all new regions, as well as all descendant records.

**Comment by stepps00** [comment]
Source: https://github.com/whosonfirst/whosonfirst-sources/blob/master/sources/svn-sma.json

File: data/101/791/993/101791993.geojson [hunk 3]
```diff
@@ -89,13 +89,13 @@
             "continent_id":102191581,
             "country_id":85633779,
             "locality_id":101791993,
-            "region_id":85677537
```
Comment (nvkelso) [inline data/101/791/993/101791993.geojson]: Why doesn't this one get a new `localadmin` in the hierarchy?

File: data/101/792/085/101792085.geojson [hunk 4]
```diff
@@ -230,13 +230,13 @@
             "continent_id":102191581,
             "country_id":85633779,
             "locality_id":101792085,
-            "region_id":85677385
```
Comment (nvkelso) [inline data/101/792/085/101792085.geojson]: Another missing `localadmin` in hierarchy.

File: data/101/792/965/101792965.geojson [hunk 5]
```diff
@@ -90,13 +90,13 @@
             "continent_id":102191581,
             "country_id":85633779,
             "locality_id":101792965,
-            "region_id":85677369
```
Comment (nvkelso) [inline data/101/792/965/101792965.geojson]: Missing `localadmin`.

File: data/101/792/971/101792971.geojson [hunk 6]
```diff
@@ -91,13 +91,13 @@
             "continent_id":102191581,
             "country_id":85633779,
             "locality_id":101792971,
-            "region_id":85677657
+            "region_id":1108959943
```
Comment (nvkelso) [inline data/101/792/971/101792971.geojson]: Missing `localadmin`.

File: data/859/297/45/85929745.geojson [hunk 7]
```diff
@@ -100,18 +103,21 @@
     "wof:geomhash":"997875718300f61b3f0fb44c4d2dd2ba",
     "wof:hierarchy":[
         {
+            "continent_id":102191581,
```
Comment (nvkelso) [inline data/859/297/45/85929745.geojson]: FTW :)

File: data/110/895/978/5/1108959785.geojson [hunk 8]
```diff
@@ -0,0 +1,65 @@
...
+    "edtf:inception":"uuuu",
+    "geom:area":0.017304,
+    "geom:bbox":"15.516935917,46.499372518,15.7722075767,46.6342559291",
+    "geom:latitude":46.563417,
+    "geom:longitude":15.627182,
+    "iso:country":"SI",
+    "lbl:latitude":46.553094,
+    "lbl:longitude":15.635838,
+    "mz:hierarchy_label":1,
+    "mz:is_current":-1,
```
Comment (nvkelso) [inline data/110/895/978/5/1108959785.geojson]: These new `region` and `localadmin` features should all be `mz:is_current` = **1**.

**Comment by nvkelso** [comment]
> 192 existing records records were given a edtf:cessation property, an mz:is_current property of 0, and a superseded link to the relevant new localadmin record that supersedes it.

These are old `county` but they're being replaced with `localadmin` records? Seems fine.

**Comment by nvkelso** [comment]
Either the missing `localadmin` noted above (there may be more) is a quick fix or file a followup issue and let's merge this one. Otherwise LGTM.

File: data/101/791/993/101791993.geojson (hunk 3 above)
Comment (stepps00) [reply data/101/791/993/101791993.geojson]: Not sure why this record (and others) didn't gain a new localadmin in the hierarchy and parent_id fields. Re-PIPing shows the correct hierarchy, though, so I'm re-PIPing all records...

```
DEBUG:root:[{u'continent_id': 102191581,
  u'country_id': 85633779,
  u'localadmin_id': 1108959719,
  'locality_id': 101791993,
  u'region_id': 1108959961}]
```

**Comment by stepps00** [comment]
Reviewer comments taken care of in 00f6292 and 282e983.

**Pull request merged**